package-lock.json

```diff
@@ -1,10487 +1,8 @@
 {
   "name": "@devoinc/browser-sdk",
-  "version": "3.0.3",
-  "lockfileVersion": 2,
+  "version": "3.0.4",
+  "lockfileVersion": 1,
```

Comment:
Is this correct @Worvast ?

Reply:
yes we need it to pass test in v12

Reply:
:-1: to downgrade the lockfile version.

I doubt you need that. Make sure you do `npm i -g npm@latest` or `npm i -g npm@7` as part of (at the beginning of) your builds, and you'll have `lockfileVersion` 2 even when using Node.js 12.

Reply:
when i create directly for v2 fail in the tests, here and in local, if i update it from v1 works, look like its ok now i hope works the actions

Reply:
But, the package.json has:
```
"engines": {
    "node": ">=14",
    "npm": ">=7"
  }
```

Reply:
Yes, @lbermudez. @Worvast changed it back: 31cd224e134f5c99fba09fe544e009d62b77273d